src/System.Security.Permissions/src/System/Security/CodeAccessPermission.cs

```diff
@@ -0,0 +1,20 @@
+namespace System.Security
+{
+    public abstract partial class CodeAccessPermission : System.Security.IPermission, System.Security.ISecurityEncodable, System.Security.IStackWalk
+    {
+        protected CodeAccessPermission() { }
+        public void Assert() { }
+        public abstract System.Security.IPermission Copy();
+        public void Demand() { }
+        public void Deny() { }
+        public override bool Equals(object obj) { return default(bool); }
+        //    public abstract void FromXml(System.Security.SecurityElement elem);
+        public override int GetHashCode() { return default(int); }
```

Comment:
For overrides on unsealed types my gut says that declaring the override to just return the base call is the best bet; so that any derived types calling base don't get misaligned between core and desktop.  (Though I guess that mainly falls on the ref, depending on the runtime's behavior of a missing `base.` invocation)